requirements/dev.in

```diff
@@ -1,7 +1,7 @@
 jupyter
 ipython
 pip-tools
-
+pre-commit
```

Comment:
need to run `make pip-compile` after adding a dependency.

Reply:
I'll do so now @cragwolfe